validator_client/src/cli.rs

```diff
@@ -306,9 +306,19 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                          for payload construction, where a strict fee recipient check will still be applied.")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("builder-registration-pubkey-override")
+                .long("builder-registration-pubkey-override")
+                .value_name("BUILDER_REGISTRATION_PUBKEY_OVERRIDE")
```

Comment:
```suggestion
                .value_name("PUBLIC_KEY")
```

We generally use the `value_name` as a sort of type hint. E.g., `INTEGER`.